packages/odyssey-react-mui/src/TagList.tsx

```diff
@@ -30,7 +31,13 @@ const TagList = ({ children }: TagListProps) => {
   );
 
   return (
-    <Stack component="ul" direction="row" spacing={2}>
+    <Stack
+      component="ul"
+      direction="row"
+      spacing={2}
+      useFlexGap
+      sx={{ flexWrap: isWrappable ? "wrap" : "nowrap" }}
```

Comment:
Can we memoize this style?

```ts
const stackStyles = useMemo(() => ({ flexWrap: isWrappable ? "wrap" : "nowrap" }), [isWrappable])
```

But even better, do we have a way of modifying this style from the `components.tsx` file where we do all our other component styles? You can grab `ownerState` from it, and we only need to pass `isWrappable` to `Stack`.

But I'm not sure if that works. Shred typically does stuff like this, but all those props are already on the component.